### PR TITLE
doc/dev/cephadm: cephadm bootstrap --shared_ceph_folder

### PR DESCRIPTION
**Pull request by sebastian-philipp**
Signed-off-by: Sebastian Wagner <sewagner@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/dev/cephadm/developing-cephadm.rst [hunk 1]
```diff
@@ -87,6 +88,26 @@ When you're done, you can tear down the cluster with::
...
 
+cephadm bootstrap --shared_ceph_folder
+======================================
+
+Cephadm can also be used directly without compiled ceph binaries.
+
+Run cephadm like so::
+
+  sudo ./cephadm bootstrap --mon-ip 127.0.0.1 \
+    --ssh-private-key /home/<ser>/.ssh/id_rsa \
```
Comment (adk3798) [inline doc/dev/cephadm/developing-cephadm.rst:99]:
```suggestion
    --ssh-private-key /home/<user>/.ssh/id_rsa \
```

File: doc/dev/cephadm/developing-cephadm.rst [hunk 2]
```diff
@@ -87,6 +88,26 @@ When you're done, you can tear down the cluster with::
...
+    --skip-mon-network \
+    --skip-monitoring-stack --single-host-defaults \
+    --skip-dashboard \ 
+    --shared_ceph_folder /home/<user>/path/to/ceph/
+
+- ``~/.ssh/id_rsa`` is used as the cluster key.  It is assumed that
+  this key is authorized to ssh with no passphrase to root@`hostname`.
+
+Source code changes made in the ``bybind/mgr/`` directory then
```
Comment (adk3798) [inline doc/dev/cephadm/developing-cephadm.rst:108]:
```suggestion
Source code changes made in the ``pybind/mgr/`` directory then
```

**Review by jmolmo — APPROVED** [review]

File: doc/dev/cephadm/developing-cephadm.rst [hunk 3]
```diff
@@ -87,6 +88,26 @@ When you're done, you can tear down the cluster with::
   sudo ../src/ckill.sh   # or,
   sudo ../src/cephadm/cephadm rm-cluster --force --fsid `cat fsid`
 
+cephadm bootstrap --shared_ceph_folder
```
Comment (liewegas) [inline doc/dev/cephadm/developing-cephadm.rst:91]: ``--shared-ceph-source`` ?

**Comment by sebastian-philipp** [comment]
`Gateway Error for url: https://files.pythonhosted.org/packages/4e/78/56aa1b5f4d8ac548755ae767d84f0be54fdd9d404197a3d9e4659d272348/setuptools-57.0.0-py3-none-any.whl`

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
jenkins test docs

**Comment by sebastian-philipp** [comment]
reopened in order to restart readthedocs

**Pull request merged**